### PR TITLE
Feature: add Expired session modal

### PR DESCRIPTION
**Pull request by josadcha**

### CODE REVIEW

**Comment by pkucherenk0** [comment]
We have interesting behavior that influence auto logout flow, when we are register new account and see confirm email screen, user already have an open session and after log inactivity we will also see this modal window.
Selection_309
We have to avoid showing it there

**Comment by pkucherenk0** [comment]
Approved

File: src/components/ExpiredSessionModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    handleSubmitExpSessionModal: () => void;
+    handleChangeExpSessionModalState: () => void;
+}
+
+export class ExpiredSessionModal extends React.Component<ExpiredSessionModalProps> {
+    public render() {
+        const {
+            title,
+            buttonLabel,
+        } = this.props;
```
Comment (akhlopiachyi) [inline src/components/ExpiredSessionModal/index.tsx]: write in one line

File: src/routes/Layout/Layout.test.tsx [hunk 2]
```diff
@@ -23,8 +25,18 @@ const defaults: LayoutProps = {
     history: createBrowserHistory(),
 };
 
+// tslint:disable: no-shadowed-variable
```
Comment (akhlopiachyi) [inline src/routes/Layout/Layout.test.tsx]: remove

File: src/routes/Layout/index.tsx [hunk 3]
```diff
@@ -208,9 +222,35 @@ class LayoutComponent extends React.Component<LayoutProps> {
...
+            handleSubmitExpSessionModal={this.handleSubmitExpSessionModal}
+        />
+    );
+
+    private handleChangeExpSessionModalState = () => {
+        this.setState({
+            isShownExpSessionModal: !this.state.isShownExpSessionModal,
+        });
+    };
+
```
Comment (akhlopiachyi) [inline src/routes/Layout/index.tsx]: remove empty line

File: src/routes/Layout/Layout.test.tsx (hunk 2 above)
Comment (josadcha) [reply src/routes/Layout/Layout.test.tsx]: the shadowed variable exist

**Pull request merged**